src/main/scala/com/lunatech/pi/Futures.scala

```diff
@@ -78,7 +88,7 @@ object Main {
     // Alternatively, and equivalent to the previous, but with reduced pedagogic value:
     //    val piF: Future[BigDecimal] = piChunks map { chunks => chunks.sum }
     // or, shorthand
-    val piF: Future[BigDecimal] = piChunks map {_.sum }
+    val piF: Future[BigDecimal] = piChunks map { _.sum }
```

Comment:
```suggestion
    val piF: Future[BigDecimal] = piChunks.map { _.sum }
```
Just realizing I missed this one, but I'd recommend against using infix operators in this case. I try to avoid infix operators that aren't math or the likes. I think having some methods using infix and others not just causes confusion.